docs/features/map-tiles-custom-areas/requirements.md

```diff
@@ -0,0 +1,11 @@
+# Role base access control - Brief
+
+This document represents the initial needs for map tiles needed at project and scenario creation that are not already defined in [this document](https://docs.google.com/document/d/1T1Gp9RM6J-CoZoyIL7XBeUU1_w2ZrvN7rCSN_XAnBSg/edit#heading=h.51ul83hfhytz).
+
+## User requirements
+
+User needs to be able to:
```

Comment:
```suggestion
All users need to be able to:
```
I'd say this could include viewers - they won't be able to create a project or a scenario, or even to upload a custom planning area or grid, for that matter, but I would say we can still keep to the pattern used so far where only post/put/patch/delete ops are restricted from viewers - it won't make any difference here in practice but I'd stick to the same pattern nevertheless (up to you, in any case).